content/en/storage-providers/setup/initialize.md

```diff
@@ -79,7 +79,7 @@ systemctl start lotus-miner
 ```
 
 {{< alert icon="warning" >}}
-**Do not proceed** from here until you have verified that your miner not only is running, but also [reachable on its public IP address]({{< relref "connectivity" >}}).
+**Do not proceed** from here until you have verified that your miner not only is running, but also [reachable on its public IP address]({{< relref "../operate/connectivity" >}}).
```

Comment:
I think full paths have already been added to the setup/initialze page, so the changes made here seems to come from an older commit maybe?